src/sentry/eventtypes/base.py

```diff
@@ -21,6 +23,13 @@ def get_title(self, metadata):
     def get_location(self, metadata):
         return None
 
+    def build_search_message(self, message, metadata):
```

Comment:
nit, please use the same naming convention of the other methods. get_search_message.

But most important, the fact you are passing in the event message to this method makes me think this may not be the right place to add this. I would get a review from someone who is more familiar with the goal of the eventtypes data structure.

Reply:
That is a good point. I'll consolidate the message generation further so that it is less spread out across `EventManager` and the eventtypes.